charts/kserve-resources/values.yaml

```
@@ -57,9 +57,167 @@ kserve:
+      imagePullSecrets: []
+      securityContext:
+        allowPrivilegeEscalation: false
+        privileged: false
+        runAsNonRoot: true
+        capabilities:
+          drop:
+            - ALL
+    sklearnserver:
+      disabled: false
+      image: kserve/sklearnserver
+      tag: *defaultVersion
```

YAMLlint (1.38.0)

[error] found undeclared alias "defaultVersion"

(anchors)